common/buildings/00_tradepost.txt

```diff
@@ -463,3 +463,301 @@ trade_post = {
+			NOT = { owner = { trait = creature_night_elf } }
+		}
+		upgrades_from = wc_goblinGhetto		
+		gold_cost = 750
+		build_time = 750
+		retinuesize = 150
+		garrison_size = 2
+	}
+	
+		
+	#RestingLounge
+	wc_restingLounge = {
+		desc = wc_restingLounge_desc
+		potential = {
+			always = yes
+			owner = { trait = creature_night_elf }
+			owner = { trait = creature_high_elf }
+			owner = { trait = creature_blood_elf }
+		}
+		gold_cost = 600
+		build_time = 500
+		local_tax_modifier = 0.5
+		garrison_size = 1
+		monthly_character_piety = 0.5
+		ai_creation_factor = 100
+	}
+	
+	#Great Clan Tent
+	wc_greatClanTent = {
+		desc = wc_greatClanTent_desc
+		potential = {
+			always = yes
+			owner = { trait = creature_orc }
+		}
+		gold_cost = 300
+		build_time = 500
+		local_tax_modifier = 0.2
+		monthly_character_piety = 0.3
+		ai_creation_factor = 100
+	}
+	
+	#Kodo Breeder
+	wc_kodoBreeder = {
+		desc = wc_kodoBreeder_desc
+		potential = {
+			always = yes
+			owner = { trait = creature_tauren }
+		}
+		gold_cost = 500
+		build_time = 500
+		trade_route_wealth = 0.2
+		local_movement_speed = 1
+		ai_creation_factor = 100
+	}
+	
+	#MerchantVenue
+	wc_merchantVenue = {
+		desc = wc_merchantVenue_desc
+		potential = {
+			always = yes
+			owner = { trait = creature_human }
+		}
+		prerequisites = { wc_bank }		
+		gold_cost = 350
+		build_time = 650
+		retinuesize = 100
+		garrison_size = 1
+	}
+	
+	#Jewelcrafters
+	wc_jewelcrafters = {
+		desc = wc_jewelcrafters_desc
+		potential = {
+			always = yes
+			port = no
+			OR = {
+			owner = { trait = creature_human }
+			owner = { trait = creature_blood_elf }
+			owner = { trait = creature_goblin }
+			}
+		}	
+		gold_cost = 750
+		build_time = 650
+		tax_income = 0.5
+	}
+	
+	#Quel'dorei Jewelcrafters
+	wc_queldoreijewelcrafters = {
+		desc = wc_queldoreijewelcrafters_desc
+		potential = {
+			always = yes
+			port = no
+			owner = { trait = creature_blood_elf }
+		}	
+		upgrades_from = wc_jewelcrafters		
+		gold_cost = 1000
+		build_time = 1050
+		tax_income = 1
+		monthly_character_prestige = 1
+	}
+	
+	#MerchantDistrict
+	wc_merchantDistrict = {
+		desc = wc_merchantDistrict_desc
+		potential = {
+			always = yes
+			owner = { trait = creature_human }
+		}
+		upgrades_from = wc_merchantVenue		
+		gold_cost = 650
+		build_time = 750
+		retinuesize = 150
+		garrison_size = 2
+	}	
+	
+	
+	#CoastalCannons
+	wc_cannons = {
+		desc = wc_cannons_desc
+		potential = {
+			always = yes
+		}	
+		gold_cost = 650
+		build_time = 300
+		fort_level = 1
+	}
+	
+	#CoastalForts
+	wc_coastalforts = {
+		desc = wc_coastalforts_desc
+		potential = {
+			always = yes
+		}
+		upgrades_from = wc_cannons		
+		gold_cost = 550
+		build_time = 250
+		fort_level = 1
+		retinuesize = 100
+		local_tax_modifier = -0.05
+	}
+	
+	#Mailboxes
+	wc_mailBox = {
+		desc = wc_mailBox_desc
+		potential = {
+			always = yes
+		}	
+		gold_cost = 350
+		build_time = 250
+		vassal_opinion = 5
+		tech_growth_modifier = 0.2
+		commander_limit = 1
```

Comment:
What if I have 50 trade posts?

Reply:
Again only 3 trade posts per ruler that arent in merchant republic

Reply:
Done! Removed entirely